bedrock/firefox/templates/firefox/faq.html

```diff
@@ -62,10 +62,10 @@ <h2>{{ ftl('what-is-firefox') }}</h2>
         <h2>{{ ftl('how-do-i') }}</h2>
         <p>{{ ftl('you-can-easily',
               url=url('firefox.new'),
-              url2=url('firefox.windows'),
-              url3=url('firefox.mac'),
-              url4=url('firefox.linux'),
-              url5=url('firefox.mobile.index')
+              url2=url('firefox.new.protocol.download_windows'),
```

Comment:
This page is throwing a 500 error for me with `Reverse for 'firefox.new.protocol.download_windows' not found.`